addons/mail/static/src/models/locale/locale.js

```diff
@@ -23,6 +31,9 @@ function factory(dependencies) {
     }
 
     Locale.fields = {
+        language: attr({
+            compute: '_computeLanguage',
+        }),
```

Comment:
content of field not clear, please document the format (is it iso code 2 char? 3 char? full name? ...)